app/client.test.js

```diff
@@ -26,6 +26,30 @@ test("instance creation and initialization", () => {
   equal(progress.finished, false)
 })
 
+test("url generate a valid URL with filters", () => {
+  // @ts-ignore
+  const client = new Client("https://example.com/", () => {})
+  const filters = {
+    count: 1,
+    sortBy: "DateAndTime",
+    sortOrder: "descending"
+  }
```

Comment:
Please add a type for the `filters` object and add all of the possible properties.

Reply:
> Please add a type for the `filters` object and add all of the possible properties.

I've added all the filters [listed here](https://api.onlyoffice.com/docspace/filters).